### PR TITLE
Task3

### PR DESCRIPTION
**Pull request by Anton-cod**
Task3

### CODE REVIEW

File: Tasks/Sushchynskiy_Tasks/Task3/Task3.py [hunk 1]
```diff
@@ -0,0 +1,54 @@
...
+        while words:
+            word = words[0]
+            words = words[1:]
+            if current_symbols_in_words(current_string + [word]) <= max_length:
+                if "\n" in word:
+                    a, b = word.split("\n")
+                    word = a
+                    words = [b] + words
+
+                    new_text += format_string(current_string + [word, "\n"])
```
Comment (MikitaTsiarentsyeu) [inline Tasks/Sushchynskiy_Tasks/Task3/Task3.py:41]: Adds an "empty" line in your file.

**Review by MikitaTsiarentsyeu — COMMENTED** [review]
Good job, your score is 9.0.

**Comment by MikitaTsiarentsyeu** [comment]
Mr. Lebedev has the same function and the the same problem with empty lines. Was it a team effort?

**Comment by Anton-cod** [comment]
Yes, something like that

**Pull request merged**